app/views/world/index.html.erb

```diff
@@ -18,7 +18,16 @@
           },
           name: "search-box",
           id: "filter-list",
-          controls: "search_results"
+          controls: "search_results",
+          data: {
+            module: "ga4-focus-loss-tracker",
+            ga4_focus_loss: {
+              event_name: "filter",
+              type: "filter",
+              action: "filter",
+              section: @presented_index.title
```

Comment:
Can we also force this to use the English translation string?

Reply:
@andysellick I don't think so as it seems to come from the content item:

`app/controllers/world_controller.rb`

```ruby
class WorldController < ApplicationController
  def index
    index = WorldIndex.find!("/world")
    @presented_index = WorldIndexPresenter.new(index)
    setup_content_item_and_navigation_helpers(index)
  end
end
```

Reply:
Okay, thanks for checking 👍